presto-docs/src/main/sphinx/release/release-0.272.rst

```diff
@@ -0,0 +1,55 @@
+=============
+Release 0.272
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix parameter ordering for prepared statement queries using a WITH clause.
+* Improve performance by enabling :doc:`/optimizer/cost-based-optimizations` by default.
+* Add :func:`secure_random()` function to return a cryptographically secure random number.
+* Add ability to stream data for partial aggregation instead of building hash tables.
+    It improves the performance of aggregation when the data is already ordered by the group-by keys.
+    This can be enabled with the ``streaming_for_partial_aggregation_enabled`` session property or the ``streaming-for-partial-aggregation-enabled`` configuration property.
+* Add an adaptive stage scheduling policy that switches to phased execution mode once a query's stage count exceeds a configurable upper bound.
+    This can be enabled by setting the session property ``execution_policy`` to ``phased`` and the stage count limit can be configured by the session property ``max_stage_count_for_eager_scheduling``.
+
+Hive Changes
+____________
+* Fix offset calculation in Parquet writer.
+* Add ability to do streaming aggregation for Hive table scans to improve query performance with aggregation when group-by keys are the same as order-by keys.
+    Cases where group-by keys are a subset of order-by keys can't enable streaming aggregation for now.
+    This can be enabled with the ``streaming_aggregation_enabled`` session property or the ``hive.streaming-aggregation-enabled`` configuration property.
+* Add ability to disable splitting file in Hive connector.
+    This can be disabled with the ``file_splittable`` session property or the ``hive.file-splittable`` configuration property.
+* Add support for using Parquet page-level statistics to skip pages.
+    This feature can be enabled by setting the ``hive.parquet-column-index-filter-enabled`` configuration property.
```

Comment:
Do these indentations matter when we render the release note?

Reply:
Yes, it indents the text (and not by using spaces, the HTML is marked up)

<img width="1136" alt="image" src="https://user-images.githubusercontent.com/13244874/161656285-5399ef3b-7b43-434a-a7f9-45789c748f2f.png">